### DISCUSSION TITLE
Notification to mark an episode as read

### DISCUSSION THREAD

**Discussion by hanneskt** [trakt/showly#521] [Ideas / Feature Requests]
One of my main uses of this app is when watching a show on my phone, and now I have to switch back and forth after every episode to mark it as watched.

It would be awesome if there was some way to activate a notification for a certain show with a button to mark it as watched. This would mean I could just pull down the notification tray and press a button on the notification instead of quitting the Netflix (or whatever) app to switch to showly and back again.

How I see this is a notification with the show title, season and episode and then just a checkmark icon like currently used to mark a show as read. When you click it it would do the same as it would clicking the checkmark next to a show in the app, and then show a new notification for the next episode (or just update the current one). Swiping it away would cancel this loop so it's easy to hide when you have finished.

The menu that shows up when you press and hold a show might be a good location for an option like this. Something like "pin to notifications".

**Comment by michaldrabik** [comment]
Interesting idea but I think it's veeeery "specific". I mean I don't really think this would have been used by more than fraction of users since you can simply open the app after you are done watching and mark everything you have just seen.
